src/test/kotlin/com/amazon/opendistroforelasticsearch/indexstatemanagement/TestHelpers.kt

```diff
@@ -54,14 +54,14 @@ import java.time.ZoneId
 import java.time.temporal.ChronoUnit
 
 fun randomPolicy(
-    name: String = ESRestTestCase.randomAlphaOfLength(10),
```

Comment:
Should this be description instead of id?

Reply:
No, this was being used as Id before it Id is correct.